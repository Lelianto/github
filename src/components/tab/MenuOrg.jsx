import React from 'react';

const MenuOrg = (props) => {
	return (
		<>
			<div className={props.scroll ? 'bg-white container flex shadow-sm fixed left-0 top-0 right-0 z-2000' : 'bg-white flex shadow-sm'}>
				<div className="w-full flex max-w-full overflow-x-scroll no-scrollbar lg:ml-14">
					<div className="flex justify-around">
						{props.menus}
					</div>
				</div>
			</div>
		</>
	)
}

export default MenuOrg;