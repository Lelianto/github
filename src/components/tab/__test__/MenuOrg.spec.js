import MenuOrg from "../MenuOrg";
import { shallow } from "enzyme";
import { BrowserRouter as Router } from 'react-router-dom';
import renderer from 'react-test-renderer';

describe("Menu Tab in Organization Page: Check content", () => {
	let wrapper;

	beforeEach(() => {
		wrapper = shallow(<MenuOrg scroll={true} menus="Overview" />)
	});

	it('Renders correctly', () => {
		const tree = renderer.create(<Router><MenuOrg scroll={true} menus="Overview" /></Router>).toJSON();
		expect(tree).toMatchSnapshot();
	});
})