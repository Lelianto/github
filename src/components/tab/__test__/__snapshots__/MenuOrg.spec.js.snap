// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu Tab in Homepage: Check content Renders correctly 1`] = `
<div
  className="bg-white sm:w-full lg:container flex shadow-sm fixed left-0 top-0 right-0 z-2000"
>
  <div
    className="sm:w-0 md:w-1/4"
  />
  <div
    className="sm:w-full md:w-1/2 flex max-w-full overflow-x-scroll no-scrollbar"
  >
    <div
      className="flex justify-around"
    >
      Overview
    </div>
  </div>
  <div
    className="sm:w-0 md:w-1/4"
  />
</div>
`;

exports[`Menu Tab in Organization Page: Check content Renders correctly 1`] = `
<div
  className="bg-white container flex shadow-sm fixed left-0 top-0 right-0 z-2000"
>
  <div
    className="w-full flex max-w-full overflow-x-scroll no-scrollbar lg:ml-14"
  >
    <div
      className="flex justify-around"
    >
      Overview
    </div>
  </div>
</div>
`;
